### ISSUE TITLE
Unable to reproduce

### ISSUE THREAD

**Issue by silvioricardoc** [ThilinaRajapakse/simpletransformers#1338]
**Describe the bug**
I am training a `MultiLabelClassificationModel` with `MultiLabelClassificationArgs(..., manual_seed=5)`. I am using `bert-base-multilingual-cased` as the language model, and I have a small dataset of around 1000 training entries for a total of around 50 different labels.

Before the first epoch, if I call `model.predict`, I get some correct predictions and a lot of bad predictions (for a low F1-score of around 0.05, which is consistent with a random initialization). After letting it run for around 5 epochs, one of these two things happens : 

1) The model starts to converge, I see overall lower running losses, and I get an F1-score of around 0.40.
2) The model stops predicting any label (so precision=recall=0). The running loss is as high as in the first epoch, it does not decrease over time.

**To Reproduce**
Train two classifiers with a given `manual_seed` and a pretrained model. Sometimes it will converge and produce reasonable results. Sometimes it will stop predicting any label at all.

As for the configurations used, convergence has been seen with the following configurations :
* `batch_size=8` and `learning_rate=1e-3`
* `batch_size=1` and `learning_rate=5e-6`

Other configurations seem to always produce a classifier that does not predict any label, but it's unclear if convergence is just rare or impossible.

**Expected behavior**
Consistency for a given `manual_seed` (with reproducible results), and a reduction of running losses over time.

**Desktop (please complete the following information):**
 - Docker image `python:3.7` (aka `3.7.12-bullseye`), ultimately a Debian system
 - Hosting operating system: Linux Mint 20.1
